docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -216,42 +228,103 @@ MetricFamilies of type Info MUST have an empty Unit string.
+
+As the Standard Schema allows for many positive and negative native buckets, the Histogram SHOULD have strategies to reset itself to empty when the number of native buckets is too high. Such strategies are out of scope for this specification.
+
+In case of Standard Schemas, the boundaries of a positive or negative native bucket with index i MUST be calculated as follows (using Python syntax):
+
+The upper inclusive limit of a positive native bucket: `(2**2**-n)**i`
+
+The lower exclusive limit of a positive native bucket: `(2**2**-n)**(i-1)`
+
+The lower inclusive limit of a negative native bucket: `-((2**2**-n)**i)`
+
+The upper exclusive limit of a negative native bucket: `-((2**2**-n)**(i-1))`
+
+i is an integer number that MAY be negative.
+
+There are exceptions to the rules above concerning the largest and smallest finite values representable as a float64 (called MaxFloat64 and MinFloat64 in the following) and the positive and negative infinity values (+Inf and -Inf):
+
+The positive native bucket that contains MaxFloat64 (according to the boundary formulas above) has an upper inclusive limit of MaxFloat64 (rather than the limit calculated by the formulas above, which would overflow float64).
+
+The next positive native bucket (index i+1 relative to the bucket from the previous item) has a lower exclusive limit of MaxFloat64 and an upper inclusive limit of +Inf. (It could be called a positive native overflow bucket.)
+
+The negative native bucket that contains MinFloat64 (according to the boundary formulas above) has a lower inclusive limit of MinFloat64 (rather than the limit calculated by the formulas above, which would underflow float64).
+
+The next negative native bucket (index i+1 relative to the bucket from the previous item) has an upper exclusive limit of MinFloat64 and an lower inclusive limit of -Inf. (It could be called a negative native overflow bucket.)
+
+Native buckets beyond the +Inf and -Inf buckets described above MUST NOT be used.
+
+If the zero native bucket is present, the Historam MetricPoint MUST have a Zero threshold. The Zero threshold MUST be a non-negative float64 value (threshold >= 0.0). The boundaries of the zero native bucket are `[-threshold, threshold]` inclusively.
+
+If the zero native bucket is present, any measured value that falls into the zero native bucket MUST be counted towards the zero native bucket and MUST NOT be counted in any other native bucket. The Zero threshold SHOULD be equal to a lower limit of an arbitrary native bucket.
+
+If the NaN value is not allowed, then the Count value MUST be equal to the sum of the negative, positive and zero native buckets.
+
+If the NaN value is allowed, it MUST NOT be counted in any bucket and the Count MUST be greater than the sum of the negative, positive and zero native buckets. The ratonale is that NaN does not belong to any bucket mathematically, and instrumentation libraries traditionally don't put it into any bucket.
```

Comment:
```suggestion
If the NaN value is allowed, it MUST NOT be counted in any bucket and the Count MUST be greater than the sum of the negative, positive and zero native buckets. The rationale is that NaN does not belong to any bucket mathematically, and instrumentation libraries traditionally don't put it into any bucket.
```

Beyond typos: Not sure if we can talk about a tradition already. Not counting NaN observations in any bucket was part of the nascent spec pretty early. So it didn't just happen out of tradition.

"Count MUST be greater than the sum..." → that's only the case if there are NaN observations in the histogram. Maybe use a wording that implies that the difference between Count and the sum of all buckets MUST be equal to the number of NaN observations.